inc/saiport.h

```diff
@@ -226,6 +226,9 @@ typedef enum _sai_port_fec_mode_extended_t
  */
 typedef enum _sai_port_priority_flow_control_mode_t
 {
+    /** Disable PFC for both rx and tx */
+    SAI_PORT_PRIORITY_FLOW_CONTROL_MODE_DISABLED,
```

Comment:
FYI, attributes are added in the middle of the enum will cause to shift other attributes and not be binary backward compatible

Reply:
move to last

Reply:
for the attribute, i think we need to explicitly add the enum value as we have discussed @kcudnik it helps to group enum together meanwhile maintain backward compatibility.

Reply:
Make sense, should I also reorder other attributes to the end of the enum. But I think it will break a kind of readability.